src/App.jsx

```diff
@@ -1,9 +1,63 @@
+import { Route, Routes, json, useNavigate } from 'react-router-dom';
 import './App.css';
+import Menu from './Menu';
+import ContactList from './ContactList';
+import CreateContact from './CreateContact';
+import Contact from './Contact';
+import { useEffect, useState} from 'react';
+import {Delete, PostContactAPI, Update } from './ConnectToAPI';
+import UpdateContact from './UpdateContact';
 
 function App() {
+
+    const [contacts, setcontacts] = useState([])
+    const navigate = useNavigate()
+     
+    useEffect(() => {
+        console.log("Fetching data")
+        fetch("https://boolean-api-server.fly.dev/Annemoon-de-Groen/contact", {}).then((response) =>{
+            return response.json();
+        }).then((jsonData) =>{
+            setcontacts(jsonData)
+        })
+    }, [])
+    
+
+
+    const addContact = (newContact) =>{
+        contacts.push(newContact)
```

Comment:
you should add to local state ONLY if the API request was successfull, becuase if there's errors you will end up with inconsistent state taht doesnt' match the data on the server